src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Deadline class which is a subclass of Event.
```

Comment:
perhaps this can be more descriptive? for example "This class encapsulates a deadline that contains..."
same for Event and ToDo